UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -544,29 +544,12 @@ extension CollectionViewModelDataSource: UICollectionViewDataSource {
 
         let invalidatedViewModelCache = invalidateViewModelCache(for: updates)
 
-        // If the collection view is not part of the window hierarchy or the application is in the background,
-        // then just do a basic reload. This avoids potential exceptions inside `UICollectionView` when a batch
-        // update spans the view being added to the hierarchy and avoids core animation queuing up animations
-        // of changes from when the app was in the background.
-        if collectionView.window == nil || inBackground {
-            // Disable animations during background/offscreen reload.
-            CATransaction.begin()
-            CATransaction.setDisableActions(true)
-
-            // CollectionView.reloadData does not synchronously fetch new information from the data source, so
-            // don't call performBatchUpdates until it does.
-            // NOTE: this path does not fire any observable events
-            collectionViewState = .loading
-            collectionView.reloadData()
```

Comment:
This was essentially a legacy workaround that was negated by the newer approach taken at line 589. Now they both go through the same codepath (and .reloadData will not necessarily dequeue new cells when not attached to a window - which is a UICV bug).